__tests__/app.test.js

```diff
@@ -132,3 +132,55 @@ describe('GET /api/articles/:article_id', () => {
       });
   });
 });
+
+describe('GET /api/articles/:article_id/comments', () => {
+  it('200: responds with all comments when given a valid article id', () => {
+    return request(app)
+      .get('/api/articles/1/comments')
+      .expect(200)
+      .then(({ body }) => {
+        const { comments } = body;
+
+        expect(comments).toHaveLength(11);
+        comments.forEach((comment) => {
+          expect(comment).toMatchObject({
+            comment_id: expect.any(Number),
+            article_id: expect.any(Number),
+            body: expect.any(String),
+            author: expect.any(String),
+            votes: expect.any(Number),
+            created_at: expect.any(String),
+          });
+        });
+      });
+  });
+
+  it('200: responds with an empty array if the article id exists but has no comments', () => {
+    return request(app)
+      .get('/api/articles/2/comments')
+      .expect(200)
+      .then(({ body }) => {
+        const { comments } = body;
+
```

Comment:
This is a really good test, Might be worth adding in an extra assertion here that checks comments is an instance of an array

Reply:
I have added the extra assertion as per your suggestion. Thanks! 😄